### ISSUE TITLE
[INFO] gentoo and github overlays

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [gentoo-science/sci#423]
In the homepage
https://wiki.gentoo.org/wiki/Project:Science/Overlay

Is written

```
We keep them both synchronized. The overlay can be accessed or installed the usual way. 
```
And the link provided is not working.
https://git.overlays.gentoo.org/gitweb/?p=proj/sci.git 

But how often they are synchronized?
~~The science overlay in layman is not up to date.~~

**Comment by jlec** [comment]
We are doing manuall sync. Currently we have no way to do it automatically. Please ping us in IRC in case they are heavily out of sync

**Comment by jlec** [comment]
https://wiki.gentoo.org/index.php?title=Project%3AScience%2FOverlay&type=revision&diff=406342&oldid=373764
